systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -1062,8 +1062,10 @@
       end
       
       model = applyToAllRigidBodyElements(model,'updateParams',fr.getPoly,p);
-
-      model = compile(model);
+      
+      if isnumeric(p)
+        model = compile(model);
+      end
```

Comment:
When running this:
r1 = RigidBodyManipulator('tensionWParams.urdf'); %tensionWParams.urdf is a parameterized urdf model

q=msspoly('q',nq);
s=msspoly('s',nq);
c=msspoly('c',nq);
qt=TrigPoly(q,s,c);
qd=msspoly('qd',nq);
qdd=msspoly('qdd',nq);
u=msspoly('u',nu);
p=r1.getParamFrame.getPoly;
pr1 = setParams(r1,p);

get this error when runnin 'pr1 = setParams(r1,p);':

Error using constructModelmex
mxGetPr can only be called on arguments which correspond to Matlab doubles

Error in RigidBodyManipulator/createMexPointer (line 2084)
        obj.mex_model_ptr = constructModelmex(obj);

Error in RigidBodyManipulator/compile (line 818)
      model = createMexPointer(model);

Error in RigidBodyManipulator/setParams (line 1067)
        model = compile(model);

Error in extractDynParams (line 16)
pr1 = setParams(r1,p);